WalletWasabi/WabiSabi/Backend/Banning/CoinVerifier.cs

```diff
@@ -21,12 +21,12 @@ public CoinVerifier(CoinJoinIdStore coinJoinIdStore, CoinVerifierApiClient apiCl
 		WabiSabiConfig = wabiSabiConfig;
 	}
 
-	// Blank constructor used for testing
+	// Constructor used for testing
 	internal CoinVerifier(CoinJoinIdStore coinJoinIdStore, CoinVerifierApiClient apiClient, WabiSabiConfig wabiSabiConfig)
 	{
 		CoinJoinIdStore = coinJoinIdStore;
 		CoinVerifierApiClient = apiClient;
-		Whitelist = new();
+		Whitelist = new(Enumerable.Empty<Innocent>(), string.Empty, new WabiSabiConfig() { ReleaseFromWhitelistAfter = TimeSpan.FromSeconds(1) });
```

Comment:
Shouldn't we just pass the `wabiSabiConfig` to the `Whitelist`

and set `ReleaseFromWhitelistAfter = TimeSpan.FromSeconds(1)` in the test file?

Reply:
ACK